SECURITY.rst

```diff
@@ -0,0 +1,129 @@
+..
+   [important] This document is reused across Astronomer OSS Integrations projects, any changes should also be applied in the other repositories
+
+Security
+________
+
+This document contains information on how to report security vulnerabilities in Astronomer Cosmos and
+how security issues reported to the Astronomer security team are handled.
+If you would like to learn more, refer to `https://www.astronomer.io/security/ <https://www.astronomer.io/security/>`_.
+
+At Astronomer, we recognize the critical nature of security and view it as a transparent and collaborative effort.
+If you have any concern about the security of any Astronomer public repository, or believe you have uncovered a vulnerability,
+please email `oss_security@astronomer.io <mailto:oss_security@astronomer.io>`_.
+
+.. warning::
+    Due to the nature of some security vulnerabilities, do not create a GitHub issue to report a vulnerability.
+
+
+Use of Email for Vulnerability Disclosure
+=========================================
+
+Only use the OSS security email to disclose security vulnerabilities.
```

Comment:
I'd suggest to use private Github reporting exclusively (and not use email). In the foundation we were exploring it and because of multi-team organisation setup of the ASF we found it not-yet-good-for-us (mainly about automation of notifications of new issues to respective PMCs), but it should be good if you have a centralized place to receive notifications about security issues in Astronomer (and it's WAY more convenient to organise your security issue handling). 

See https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability

Reply:
@potiuk Our longer-term goal in Astronomer is to unify all vulnerability disclosures for any of our products, repos, etc, to use the Bugcrowd platform we have adopted (https://www.astronomer.io/vulnerability-disclosure/). Due more to internal organizational structure, etc, an email helps us more closely align and consolidate processes until we expand adoption of our larger VDP.

Reply:
Understood. More or less same reason why ASF have not yet moved to private Github reporting.

Reply:
Thanks, @potiuk , for the feedback and @chaosmaw, for leading this initiative in Astronomer. It's very exciting we're heading towards the same approach in the mid-term.